solidity/contracts/integrator/TBTCDepositorProxy.sol

```diff
@@ -204,6 +211,8 @@ abstract contract TBTCDepositorProxy {
         // slither-disable-next-line reentrancy-no-eth
         delete pendingDeposits[depositKey];
 
+        initialDepositAmount = deposit.amount * SATOSHI_MULTIPLIER;
```

Comment:
Let's update the usage example from the contract's docstring accordingly. Right now, there is an obsolete version with `_finalizeDeposit` returning two parameters.

Reply:
4d74ad9fa082a4e2cf0a9a91d45d5b6b03a0e556